system_tests/monitoring.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest2
+
+from gcloud import _helpers
+from gcloud.environment_vars import TESTS_PROJECT
+from gcloud import monitoring
+
+
+METRIC_TYPE = 'pubsub.googleapis.com/topic/send_message_operation_count'
+METRIC_KIND = monitoring.MetricKind.DELTA
+VALUE_TYPE = monitoring.ValueType.INT64
+
+RESOURCE_TYPE = 'pubsub_topic'
+
+
+def setUpModule():
+    _helpers.PROJECT = TESTS_PROJECT
+
+
+class TestMonitoring(unittest2.TestCase):
```

Comment:
It depends on what you mean. The current system tests don't really assume anything about the state of the project. The monitored resource descriptors and the curated metric descriptors are defined by GCP. However, if someone has defined custom metric descriptors, list_metric_descriptors() will return those too.

It'd be great to have system tests that confirm that our understanding of the API matches the actual API, or at least that we're using the interface correctly. This particular API is somewhat problematic, because it doesn't promise that you can immediately read what you have written.